src/librbd/Operations.cc

```diff
@@ -950,21 +950,7 @@ int Operations<I>::snap_protect(const char *snap_name) {
   if (r < 0) {
     return r;
   }
-
```

Comment:
Leave all of this code -- we still want the API to validate but we can improve the experience from the rbd CLI